src/modules/evas/engines/gl_common/evas_gl_core.h

```diff
@@ -8,14 +8,10 @@
 #endif
 
 #ifdef _WIN32
-# ifdef EFL_BUILD
-#  ifdef DLL_EXPORT
-#   define EMODAPI __declspec(dllexport)
-#  else
-#   define EMODAPI
-#  endif
+# ifndef EFL_MODULE_STATIC
+#  define EMODAPI __declspec(dllexport)
 # else
-#  define EMODAPI __declspec(dllimport)
+#  define EMODAPI
```

Comment:
Wait: we're removing the `dllimport` possibility. Is this right (I didn't work too much with modules, so I sincerely don't know)?

Reply:
@JPTIZ this was discussed in [slack](https://expertisesolutions.slack.com/archives/C7NGCQY8M/p1593524654020200?thread_ts=1593524654.020200).

Reply:
Thanks. Modules are never imported since they're loaded manually in runtime, so it makes sense.